### ISSUE TITLE
minikube ip Exiting due to GUEST_NODE_RETRIEVE: Could not find node

### ISSUE THREAD

**Issue by hayorov** [kubernetes/minikube#10322]
**Steps to reproduce the issue:** 

1. brew install minikube@1.17.1
2. minukube start
3. minikube ip

**Full output of failed command:** 
 ❯ minikube ip --v=8 --alsologtostderr
I0131 15:20:36.463309   16328 out.go:229] Setting OutFile to fd 1 ...
I0131 15:20:36.463716   16328 out.go:281] isatty.IsTerminal(1) = true
I0131 15:20:36.463723   16328 out.go:242] Setting ErrFile to fd 2...
I0131 15:20:36.463728   16328 out.go:281] isatty.IsTerminal(2) = true
I0131 15:20:36.463806   16328 root.go:291] Updating PATH: /Users/alexk/.minikube/bin
I0131 15:20:36.464359   16328 mustload.go:66] Loading cluster: minikube
I0131 15:20:36.465485   16328 main.go:119] libmachine: Found binary path at /Users/alexk/.minikube/bin/docker-machine-driver-hyperkit
I0131 15:20:36.465538   16328 main.go:119] libmachine: Launching plugin server for driver hyperkit
I0131 15:20:36.475493   16328 main.go:119] libmachine: Plugin server listening at address 127.0.0.1:53014
I0131 15:20:36.476038   16328 main.go:119] libmachine: () Calling .GetVersion
I0131 15:20:36.476924   16328 main.go:119] libmachine: Using API Version  1
I0131 15:20:36.476941   16328 main.go:119] libmachine: () Calling .SetConfigRaw
I0131 15:20:36.477191   16328 main.go:119] libmachine: () Calling .GetMachineName
I0131 15:20:36.477317   16328 main.go:119] libmachine: (minikube) Calling .GetState
I0131 15:20:36.477438   16328 main.go:119] libmachine: (minikube) DBG | exe=/Users/alexk/.minikube/bin/docker-machine-driver-hyperkit uid=0
I0131 15:20:36.477701   16328 main.go:119] libmachine: (minikube) DBG | hyperkit pid from json: 15777
I0131 15:20:36.479541   16328 host.go:66] Checking if "minikube" exists ...
I0131 15:20:36.479812   16328 main.go:119] libmachine: Found binary path at /Users/alexk/.minikube/bin/docker-machine-driver-hyperkit
I0131 15:20:36.479844   16328 main.go:119] libmachine: Launching plugin server for driver hyperkit
I0131 15:20:36.488713   16328 main.go:119] libmachine: Plugin server listening at address 127.0.0.1:53020
I0131 15:20:36.489104   16328 main.go:119] libmachine: () Calling .GetVersion
I0131 15:20:36.489532   16328 main.go:119] libmachine: Using API Version  1
I0131 15:20:36.489547   16328 main.go:119] libmachine: () Calling .SetConfigRaw
I0131 15:20:36.489866   16328 main.go:119] libmachine: () Calling .GetMachineName
I0131 15:20:36.489993   16328 main.go:119] libmachine: (minikube) Calling .DriverName
I0131 15:20:36.513440   16328 out.go:119] 

W0131 15:20:36.513594   16328 out.go:181] ❌  Exiting due to GUEST_NODE_RETRIEVE: Could not find node 
❌  Exiting due to GUEST_NODE_RETRIEVE: Could not find node 
W0131 15:20:36.515643   16328 out.go:181] 

W0131 15:20:36.515684   16328 out.go:181] 😿  If the above advice does not help, please let us know: 
😿  If the above advice does not help, please let us know: 
W0131 15:20:36.515721   16328 out.go:181] 👉  https://github.com/kubernetes/minikube/issues/new/choose
👉  https://github.com/kubernetes/minikube/issues/new/choose
I0131 15:20:36.584507   16328 out.go:119] 

**Full output of `minikube start` command used, if not already included:**
❯ minikube start
😄  minikube v1.17.1 on Darwin 11.1
🆕  Kubernetes 1.20.2 is now available. If you would like to upgrade, specify: --kubernetes-version=v1.20.2
✨  Using the hyperkit driver based on existing profile
👍  Starting control plane node minikube in cluster minikube
🔄  Restarting existing hyperkit VM for "minikube" ...
🐳  Preparing Kubernetes v1.19.0 on Docker 19.03.6 ...
🔎  Verifying Kubernetes components...
🌟  Enabled addons: storage-provisioner, default-storageclass, dashboard
🏄  Done! kubectl is now configured to use "minikube" cluster and "" namespace by default

** Environment **
 ❯ uname -a     
Darwin alexmbp 20.2.0 Darwin Kernel Version 20.2.0: Wed Dec  2 20:39:59 PST 2020; root:xnu-7195.60.75~1/RELEASE_X86_64 x86_64

Meanwhile, IP returns when the `-n` argument is defined 
 ❯ k get nodes
NAME       STATUS   ROLES    AGE     VERSION
minikube   Ready    master   2m10s   v1.19.0

❯ minikube ip -n minikube --v=8 --alsologtostderr
I0131 15:28:11.675406   17047 out.go:229] Setting OutFile to fd 1 ...
I0131 15:28:11.675947   17047 out.go:281] isatty.IsTerminal(1) = true
I0131 15:28:11.675957   17047 out.go:242] Setting ErrFile to fd 2...
I0131 15:28:11.675964   17047 out.go:281] isatty.IsTerminal(2) = true
I0131 15:28:11.676064   17047 root.go:291] Updating PATH: /Users/alexk/.minikube/bin
I0131 15:28:11.676604   17047 mustload.go:66] Loading cluster: minikube
I0131 15:28:11.677538   17047 main.go:119] libmachine: Found binary path at /Users/alexk/.minikube/bin/docker-machine-driver-hyperkit
I0131 15:28:11.677602   17047 main.go:119] libmachine: Launching plugin server for driver hyperkit
I0131 15:28:11.688985   17047 main.go:119] libmachine: Plugin server listening at address 127.0.0.1:53289
I0131 15:28:11.689705   17047 main.go:119] libmachine: () Calling .GetVersion
I0131 15:28:11.690506   17047 main.go:119] libmachine: Using API Version  1
I0131 15:28:11.690522   17047 main.go:119] libmachine: () Calling .SetConfigRaw
I0131 15:28:11.690864   17047 main.go:119] libmachine: () Calling .GetMachineName
I0131 15:28:11.691021   17047 main.go:119] libmachine: (minikube) Calling .GetState
I0131 15:28:11.691171   17047 main.go:119] libmachine: (minikube) DBG | exe=/Users/alexk/.minikube/bin/docker-machine-driver-hyperkit uid=0
I0131 15:28:11.691726   17047 main.go:119] libmachine: (minikube) DBG | hyperkit pid from json: 16705
I0131 15:28:11.693466   17047 host.go:66] Checking if "minikube" exists ...
I0131 15:28:11.693832   17047 main.go:119] libmachine: Found binary path at /Users/alexk/.minikube/bin/docker-machine-driver-hyperkit
I0131 15:28:11.693863   17047 main.go:119] libmachine: Launching plugin server for driver hyperkit
I0131 15:28:11.704383   17047 main.go:119] libmachine: Plugin server listening at address 127.0.0.1:53295
I0131 15:28:11.704877   17047 main.go:119] libmachine: () Calling .GetVersion
I0131 15:28:11.705465   17047 main.go:119] libmachine: Using API Version  1
I0131 15:28:11.705481   17047 main.go:119] libmachine: () Calling .SetConfigRaw
I0131 15:28:11.705951   17047 main.go:119] libmachine: () Calling .GetMachineName
I0131 15:28:11.706155   17047 main.go:119] libmachine: (minikube) Calling .DriverName
I0131 15:28:11.746611   17047 out.go:119] 192.168.64.3
192.168.64.3

Probably affected by multi-instance support, the latest touch was here https://github.com/kubernetes/minikube/pull/9873

**Comment by netrebel** [comment]
I started getting this error too. Workaround is ` minikube ip -n minikube`, see below:

```
➜  kubernetes git:(master) ✗ minikube version
minikube version: v1.17.1
commit: 043bdca07e54ab6e4fc0457e3064048f34133d7e

➜  kubernetes git:(master) ✗ minikube ip

❌  Exiting due to GUEST_NODE_RETRIEVE: Could not find node

😿  If the above advice does not help, please let us know:
👉  https://github.com/kubernetes/minikube/issues/new/choose

➜  kubernetes git:(master) ✗ minikube node list
minikube	192.168.64.13

➜  kubernetes git:(master) ✗ minikube ip -n minikube
192.168.64.13
```

**Comment by spowelljr** [comment]
@hayorov In your issue you're restarting an existing, I'm curious if you delete and re-start again if the issue is solved.

There could also possibly be a fix in v1.18.1 if you could download the latest version, thanks.

**Comment by spowelljr** [comment]
Hi @hayorov, we haven't heard back from you, do you still have this issue?
There isn't enough information in this issue to make it actionable, and a long enough duration has passed, so this issue is likely difficult to replicate.

I will close this issue for now but feel free to reopen when you feel ready to provide more information.
